zone/raids.cpp

```diff
@@ -1917,3 +1920,37 @@ void Raid::RaidMessage_StringID(Mob* sender, uint32 type, uint32 string_id, cons
 		}
 	}
 }
+
+void Raid::SetConsent(const char *name, bool grant_consent_to_raid_members)
+{
+	std::string query = StringFormat("UPDATE raid_members SET is_consent = %d WHERE name = '%s'",
+									grant_consent_to_raid_members, name);
+	auto results = database.QueryDatabase(query);
+	if (!results.Success())
+		Log(Logs::General, Logs::Error, "Set raid Consent error: %s\n", results.ErrorMessage().c_str());
+
+	LearnMembers();
+	VerifyRaid();
+
+	// Broadcast a group update to trigger an update for raid members in other zones.
```

Comment:
@SecretsOTheP I think I had already added cross-zone support in this PR exactly as you suggested, and it appeared to work properly in some simple testing that I did.

Can you confirm the code here is what you wanted or give me a clue about what I missed?  It sends a ServerOP_UpdateGroup with the group ID set to 12 which results in a no-op in UpdateGroup() but should still trigger the LearnGroup() database reload/refresh in the other zone servers to pick up the new is_consent state.

Reply:
Gonna be honest: I missed this functionality. LGTM.